### ISSUE TITLE
0.4.2 crash when following the Getting Started Guide.

### ISSUE THREAD

**Issue by ringods** [hashicorp/terraform#1593]
To introduce myself to Terraform, I'm following the Getting Started Guide. I ended up on adding the Elastic IP to the single machine setup:

https://terraform.io/intro/getting-started/dependencies.html

But when running `terraform apply`, I get this `crash.log`:

https://gist.github.com/ringods/233980ac33a4b5d851bb

Here is my `.tf` file so far:

```
provider "aws" {
    access_key = "MY_KEY"
    secret_key = "MY_SECRET"
    region = "us-east-1"
}

resource "aws_instance" "example" {
    ami = "ami-aa7ab6c2"
    instance_type = "t1.micro"
}

resource "aws_eip" "ip" {
    instance = "${aws_instance.example.id}"
}
```

Terraform 0.4.2 installed via Mac Homebrew on 10.10.3.

**Comment by mitchellh** [comment]
Ah, this will be fixed in the next version by #1555

**Comment by mitchellh** [comment]
Sorry!
